Allow setting job_id on cron jobs

JonasKs opened the pull request:
Closes #292 

I will add tests if you accept the feature.

sondrelg commented on arq/cron.py:
```diff
@@ -159,7 +161,8 @@ def cron(
         defaults to 123456 as the world is busier at the top of a second, 0 - 1e6
     :param run_at_startup: whether to run as worker starts
-    :param unique: whether the job should be only be executed once at each time
+    :param unique: whether the job should be only be executed once at each time (useful if you have multiple workers)
```
Think there's a "be" in here too many

JonasKs replied:
Fixed. Thanks :blush:

samuelcolvin commented:
I'm happy with this, please rebase to master and add tests.

JonasKs commented:
Rebased and added a test, @samuelcolvin.

Not entirely sure how we should test this without using burst, so I used burst + two different functions. Two cron jobs using the same function with `run_at_startup=True` would still only be run once.

samuelcolvin commented:
great, thank you.
